app/(legacy)/explore/page.tsx

```
@@ -0,0 +1,129 @@
+              metadata={{
+                data: {
+                  bio: project.fractions.data[0].metadata.description,
+                  impactCategory:
+                    project.fractions.data[0].metadata.impact_scope,
+                },
+              }}
+              isLoading={false}
+              buttonText="Fund this project"
+              buttonLink={`/fund/${project.hypercert_id}`}
+            />
+          ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)